data-esnext.js

```diff
@@ -1362,6 +1362,7 @@ exports.tests = [
         ie11: false,
         chrome1: false,
         chrome90: false,
+        chrome97: true,
```

Comment:
might be good to confirm it's false in chrome96

Reply:
I cannot find where to download Chrome 96, but it's not present in V8 9.6 (which should be the one used by Chrome 96).

Reply:
verified on browserstack.